packages/components-js/src/pages/fieldset-wrapper.html

```diff
@@ -4,19 +4,31 @@
   }
 </style>
 
-<div class="playground light" title="should render with label">
+<div class="playground light" title="should render with label on light background">
```

Comment:
Might be nice to apply the same grid similar as text-field-wrapper, textarea-wrapper and select-wrapper do it for a better overview on larger viewports.

Reply:
Done